integration-tests/gradle/src/integrationTest/kotlin/org/jetbrains/dokka/it/gradle/Collector0IntegrationTest.kt

```diff
@@ -52,6 +52,7 @@ class Collector0IntegrationTest(override val versions: BuildVersions) : Abstract
             assertNoUnresolvedLinks(file)
             assertNoHrefToMissingLocalFileOrDirectory(file)
             assertNoEmptyLinks(file)
+            assertNoEmptySpans(file)
```

Comment:
This makes a lot of sense to me. Good point ☺️

Reply:
Tried to do this in this PR, but this is not a trivial thing (since some assertions require configuration)
There are multiple tickets to work on integration tests. I think this shall be done in one of those!